cartridge-cli.lua

```diff
@@ -922,19 +919,58 @@ ENV PATH="/usr/share/tarantool/tarantool-enterprise:${"$"}{PATH}"
 -- * ---------------- Generic packing ----------------
 
 local function get_rock_versions(project_dir)
-    local query = luarocks_searcher.make_query("")
-    query.exact_name = false
+    local function read_manifest_file(filepath)
+        local res = {}
+
+        local file_content = read_file(filepath)
+        file_content = file_content:gsub("^#![^\n]*\n", "")
+
+        local chunk, ok, err
+        if _VERSION == "Lua 5.1" then -- Lua 5.1
+            chunk, err = loadstring(file_content, filepath)
```

Comment:
Are we plan to support plain "Lua 5.1"?

"load" is available in LuaJIT and Tarantool. Also loadstring is deprecated function

Reply:
https://stackoverflow.com/questions/37641153/loadstring-function-replacement-in-latest-version

Reply:
What needs for the second argument? As I understand, `loadstring` receives only one argument

Reply:
> What needs for the second argument? As I understand, `loadstring` receives only one argument

This argument improves verbosity in case of error in loadstring function

Reply:
The second argument is needed to form correct error msg.

Anyway, I don't use `loadstring` function anymore, but for `load` call I pass filename as a `source` parameter. I'm not sure, that it's right, but it's not so important.